mlrun/runtimes/base.py

```diff
@@ -674,7 +674,7 @@ def as_step(
         selector="",
         hyper_param_options: HyperParamOptions = None,
         inputs: dict = None,
-        outputs: dict = None,
+        outputs: list = None,
```

Comment:
This is a user-facing function, how will this change affect its usage?

Reply:
it won't really change anything as this is type-hinting. if a client passed a dictionary - the execution fails